site/gulpfile.js

```diff
@@ -22,6 +25,19 @@ const RESIZE_OPTS = {
     errorOnUnusedConfig: false
 };
 
+let manifest;
+if (PRODUCTION) {
+    // import rev-manifest if it exists
+    try {
+        manifest = require(REV_PATH);
+    } catch(error) { }
+} else {
```

Comment:
Can we avoid the else branch by setting it as the default value? It's possible you're in production but requiring the manifest fails -> there is no manifest

Reply:
Yes! Done